package.json

```diff
@@ -45,6 +45,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.21.0",
+    "@formio/react": "^5.2.4-rc.1",
```

Comment:
I can adjust the package but it worries me a bit, I based on the repository linked in on both packages link to the same repository but react-formio’s last update has been 2 years ago. I think it has seen a rename.

Reply:
it was indeed renamed and it has plenty of issues, but this is the dependency used in both the backend and SDK so that is what you need to align with.

> Do we want to update the sdk to @react/formio?

No, we will update the SDK to use this library you're developing. We will definitely not invest resources in something that is going to be replaced anyway.